srcCpp/perftest_publisher.cxx

```diff
@@ -1423,29 +1430,9 @@ class LatencyListener : public IMessagingCB
             cpu.initialize();
         }
 
-        printf("Length: %5d"
-               " Latency: Ave %6.0lf " PERFT_TIME_UNIT
-               " Std %6.1lf " PERFT_TIME_UNIT
-               " Min %6lu " PERFT_TIME_UNIT
-               " Max %6lu " PERFT_TIME_UNIT
-               " 50%% %6lu " PERFT_TIME_UNIT
-               " 90%% %6lu " PERFT_TIME_UNIT
-               " 99%% %6lu " PERFT_TIME_UNIT
-               " 99.99%% %6lu " PERFT_TIME_UNIT
-               " 99.9999%% %6lu " PERFT_TIME_UNIT
-               " %s\n",
-                totalSampleSize,
-                latency_ave, latency_std, latency_min, latency_max,
-                _latency_history[count*50/100],
-                _latency_history[count*90/100],
-                _latency_history[count*99/100],
-                _latency_history[(int)(count*(9999.0/10000))],
-                _latency_history[(int)(count*(999999.0/1000000))],
-                outputCpu.c_str());
-        fflush(stdout);
-
       #ifndef RTI_MICRO
-        if (_PM->get<bool>("serializationTime")) {
+        if (_PM->get<bool>("serializationTime"))
```

Comment:
I believe that being in the same line was ok:

for and ifs will have this syntax:

```
if (a == b) {

}
```

Functions however we decided will have this syntax

```
int my_function(int parameter)
{
    ...
}
```

Reply:
ok